ui/components/ui/button/button.component.js

```diff
@@ -21,10 +19,7 @@ const typeHash = {
   'danger-primary': 'btn-danger-primary',
   link: 'btn-link',
   inline: CLASSNAME_INLINE,
-  // TODO: Legacy button type to be deprecated
-  confirm: CLASSNAME_CONFIRM,
   raised: CLASSNAME_RAISED,
-  'first-time': CLASSNAME_FIRST_TIME,
 };
 
 const Button = ({
```

Comment:
Consolidating and removing unused types. `raised` is used but only once that I can find going to leave it for now and add it to stories.